packages/theming/atlas/src/theme/native/exclusion-variables.ts

```diff
@@ -80,6 +80,8 @@ export const excludeSafeAreaView = false;
 // Slider
 export const excludeSlider = false;
 export const excludeSliderHelpers = false;
+// Screenshot widgets
+export const excludeScreenshot = false;
```

Comment:
Is this something we want to deliver to clients? I understood from Danny this should be an internal widget

Reply:
Just confirmed with @cjfhodges and this widget shoulnt be released! Specially with Atlas styles

Reply:
done